src/dymaptic.GeoBlazor.Core/Components/Layers/TileLayer.cs

```diff
@@ -16,13 +18,55 @@ public class TileLayer : Layer
     /// <inheritdoc />
     [JsonPropertyName("type")]
     public override string LayerType => "tile";
+    
+    /// <summary>
+    ///     An authorization string used to access a resource or service. API keys are generated and managed in the ArcGIS Developer dashboard. An API key is tied explicitly to an ArcGIS account; it is also used to monitor service usage. Setting a fine-grained API key on a specific class overrides the global API key.
+    /// </summary>
+    [Parameter]
+    [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
+    public string? ApiKey { get; set; }
```

Comment:
Do we want to move this to the layer class?  Then it can be inherited by other layer types as they are added in the future or only insert this for specific layers as needed, like this case

Reply:
No, it's based on the inheritance pattern in ArcGIS. ApiKey is only available on some Layers.